deltacat/utils/pyarrow.py

```diff
@@ -395,6 +447,7 @@ def s3_file_to_parquet(
     column_names: Optional[List[str]] = None,
     include_columns: Optional[List[str]] = None,
     pa_read_func_kwargs_provider: Optional[ReadKwargsProvider] = None,
+    partial_file_download_params: Optional[PartialFileDownloadParams] = None,
```

Comment:
What is this used for?

Reply:
Partially download a parquet file if it's too big.